### PR TITLE
chore(trading): fix tests from i18n

### PR DESCRIPTION
**Pull request by bwallacee**
Fixes the python tests broken from the i18n PR.

### CODE REVIEW

**Review by bglownia — APPROVED** [review]

File: apps/trading/e2e/README.md [hunk 1]
```diff
@@ -100,13 +100,13 @@ poetry run pytest
...
 ```bash
-poetry run pytest -k "test_name" -s
+poetry run pytest -k "test_name" -s --headed
 ```
 
 Run from anywhere:
 
 ```bash
-yarn trading:test -- "test_name" -s
+yarn trading:test -- "test_name" -s --headed
```
Comment (dalebennett1992) [inline apps/trading/e2e/README.md:109]: do we want to encourage running in headed by default? Normally the opposite is true unless debugging. No strong opinion, just flagging

File: apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py [hunk 2]
```diff
@@ -47,5 +47,5 @@ def test_should_show_an_error_if_your_balance_is_zero(continuous_market, vega: V
     # 7002-SORD-060
     expect(page.get_by_test_id(place_order)).to_be_enabled()
     # 7002-SORD-003
-    expect(page.get_by_test_id("deal-ticket-error-message-zero-balance")).to_have_text("You need tDAI in your wallet to trade in this market. Make a deposit")
+    expect(page.get_by_test_id("deal-ticket-error-message-zero-balance")).to_have_text("You need tDAI in your wallet to trade in this market.Make a deposit")
```
Comment (dalebennett1992) [inline apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py:50]: ooh weird. How does the lack of this white space look in the ui out of interest, is it noticeable?

File: apps/trading/e2e/README.md (hunk 1 above)
Comment (bwallacee) [reply apps/trading/e2e/README.md:109]: Can revert, I just find when i'm running single tests i usually would want them headed.
can remove this and just make a note to show the command required to run in headed.

File: apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py (hunk 2 above)
Comment (bwallacee) [reply apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py:50]: Different lines so the space isn't required

Comment (dalebennett1992) [reply apps/trading/e2e/tests/deal_ticket/test_trading_deal_ticket_submit_account.py:50]: nice.

File: apps/trading/e2e/README.md (hunk 1 above)
Comment (dalebennett1992) [reply apps/trading/e2e/README.md:109]: up to you :)

**Pull request merged**